src/services-managing.rst

```diff
@@ -61,49 +61,19 @@ Before you are able to synchronize data or application files to a device, you wi
 
       If you are using SSL, be sure to specify :code:`https://...`
 
-    .. _services-login-with-qr:
-
-   ODK-X Services also provides a quick and easy option to login with a QR code.
-
-      1. Click on the QR icon in the top right corner.
-
-        .. image:: /img/services-managing/services-login-qr.* 
-          :alt: Services login with QR code
-          :class: device-screen-vertical
-          
-      2. The app will ask for camera permissions. Click :guilabel:`Allow`.
-      3. QR code scanner screen will appear. Scan a valid QR code.
-
-      .. note::
-
-        Use the web tab in the ODK-X Application Designer to generate QR codes.
-
   4. Authenticate user credentials
 
     .. note::
 
       If your server is configured to allow anonymous access this step is optional.
 
     a. Change the :guilabel:`Server Sign-on Credential` to :menuselection:`Username` and enter the appropriate credentials in the :guilabel:`Username` and :guilabel:`Server Password` fields.
-    b. Exit out of the :menuselection:`Server Settings` page, and then the :menuselection:`Settings` page, by using the back button.
-    c. You will then be asked to :guilabel:`Authenticate Credentials`. Select the :guilabel:`Authenticate New User` option.
+    b. Exit out of the :menuselection:`Server Settings` page, and then the :menuselection:`Settings` page, by using the back button and you should see that you have been logged in as an :guilabel:`Authenticated User`.
```

Comment:
This sentence is long. Can split at "and you" into two sentences (period after back button, then something like "There you should see..."